portfolio/src/main/webapp/script.js

```diff
@@ -16,13 +16,50 @@
  * Adds a random greeting to the page.
  */
 function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+    const greetings =
+        ["Hello, you look cute today. Lets be cute together!", "¡Hola Mundo! Soy Dominicano", "你好，世界！<3 ", "Bonjour le monde! Je t'aime "];
 
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+    // Pick a random greeting.
+    const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+
+    // Add it to the page.
+    const greetingContainer = document.getElementById('greeting-container');
+    greetingContainer.innerText = greeting;
+
+    scareUser();
+    askIfScared();
+    
+}
+
+function askIfScared() {
+    const sacredText = document.getElementById("scaredText");
+    scaredText.innerHTML = "Sorry... did I " + "<strong>scare</strong>".italics() + " you?";
```

Comment:
You can use string interpolation here
scaredText.innerHTML = `Sorry... did I ${'scare'.bold().italics()} you?';

Reply:
Is that a spacial syntax or character? For some reason it works when I copy and paste the ` at the beginning and add it to the end. How you have it now gives me an error because the ` and the ' do not match.
I tried the following and none of them worked:
`Sorry... did I ${'scare'.bold().italics()} you?';
'Sorry... did I ${'scare'.bold().italics()} you?';
"Sorry... did I ${'scare'.bold().italics()} you?";
'Sorry... did I ${"scare".bold().italics()} you?';

The only one that seems to work is 
`Sorry... did I ${'scare'.bold().italics()} you?`;
but I don't know how to recreate the ` character without copying and pasting.

Reply:
It's backticks (the backtick next to 1)

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals

I see I actually have a typo in my original comment :(

```javascript
scaredText.innerHTML = `Sorry... did I ${'scare'.bold().italics()} you?`;
```

Reply:
okay I see it now. Thank you!